src/backend/DTO/Offers/UpdateOfferDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace backend.DTO.Offers
+{
+    public class UpdateOfferDto
+    {
+        public string FoodName{ get; set; }
+        public string FoodUnit{ get; set; }
+        public double Quantity{ get; set; }
+        public DateTime ExpiresAt{ get; set; }
+        public string Description{ get; set; }
+    }
+}
```

Comment:
Since you didn't actually accept any files, you could have simply used a nested food object instead of the `Food<Property>` workaround but we will implement file upload, so this DTO will be correct in that case.